### PR TITLE
Feature/week1

### PR DESCRIPTION
**Pull request by ltejedajr**

### CODE REVIEW

File: students/images/me.JPG
Comment (mattchristenson) [inline students/images/me.JPG:1]: We will want to remove this file from this commit as it doesn't belong.  we'll do this as a group.  Rejecting PR for now.

**Review by mattchristenson — CHANGES_REQUESTED** [review]
need to remove me.JPG from PR

File: students/images/me.JPG
Comment (ltejedajr) [reply students/images/me.JPG:1]: Ok, will standby until then. Thanks Matt.

**Review by mattchristenson — APPROVED** [review]
looks good now

**Pull request merged**